_ml-commons-plugin/model-access-control.md

```diff
@@ -0,0 +1,600 @@
+---
+layout: default
+title: Model access control
+has_children: false
+nav_order: 180
+---
+
+# Model access control
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/ml-commons/issues/815).  
+{: .warning}
+
+You can use the Security plugin with ML Commons to limit non-admin users' ability to access specific models. For example, users of different departments in an organization might want to restrict access to their models only to users in their own department.
+
+Users can be assigned one or more [_backend roles_]({{site.url}}{{site.baseurl}}/security/access-control/index/) to configure fine-grained access to models based on [_user roles_]({{site.url}}{{site.baseurl}}/ml-commons-plugin/index#permissions). A backend role helps map a set of users to the same user role. For example, users with the `IT` backend role may be assigned the `ml_full_access` user role that grants full access to all ML Commons features.
+
+## Model groups
+
+For access control, models are organized into _model groups_---collections of versions of a particular model. Like users, model groups can be assigned one or more backend roles. All versions of the same model share the same model name and have the same backend role or roles. 
+
+You are considered a model _owner_ when you create a new model or upload a new version of a model. When a model owner creates a model group, the owner can specify one of the following _access modes_ for this model group:
+
+- `public`: All users who have access to the cluster can access this model group.
+- `private`: Only the model owner or an admin user can access this model group.
+- `restricted`: The owner, an admin user, or any user who shares one of the model group's backend roles can access any model in this model group. When creating a `restricted` model group, the owner must attach one or more of the owner's backend roles to the model. For a user to be able to access a model group, one of the user's backend roles must match one of the model group's backend roles. This grants the user permissions associated with the user role to which that backend role is mapped. For example, if the backend role is mapped to the `ml_full_access` user role, users with this role can use the Register, Deploy, Predict, Delete, Search and Get Model APIs on any model in this model group. 
```

Comment:
ml_full_access is the permission/security role actually. Also its not necessary a backend role should be mapped to these permission roles, a user can directly be mapped to these permission roles. For example, a user is direclt mapped to "ml_full_access" and has a backend role of that of a model group, then then can perform all the said actions on the models of this group. We might have to rephrase this paragraph a little.

Reply:
Thank you for the clarification, reworded.